class-components/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "class-components",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint 'src/**/*.{js,jsx,ts,tsx}' --fix",
+    "preview": "vite preview",
+    "prepare": "husky install",
+    "format": "prettier --write .",
```

```suggestion
    "format": "prettier --check .",
```
You have to check code using this script and fix issues using 'format:fix' script.